src/OAuth2Demo/Client/Controllers/RequestResource.php

```diff
@@ -34,8 +34,6 @@ public function requestResource(Application $app)
         $response = $request->send();
         $json = json_decode((string) $response->getBody(), true);
 
-        $resource_uri = sprintf('%s%saccess_token=%s', $endpoint, false === strpos($endpoint, '?') ? '?' : '&', $token);
```

Comment:
Of course the `access_token` is important, but it is already available via the config `$config['resource_params']['access_token'] = $token;` together with `url_generator`. Right now the `access_token` param is added twice (you could test it on your live example).